lib/sqlalchemy/ext/asyncio/engine.py

```diff
@@ -476,6 +476,10 @@ async def close(self) -> None:
         """
         await greenlet_spawn(self._proxied.close)
 
+    async def aclose(self) -> None:
+        """Call the close() method of :class:`_asyncio.AsyncConnection`"""
```

Comment:
**mike bayer** (zzzeek) wrote:

needs `.. versionadded: 2.0.20 ``

also add a line why we have `aclose()` and how it differs from `close()` (answer seems to be: "this is a synonym for `close()` that is compatible with context managers such as `contextlib.closing()`

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4785

Reply:
done